src/sdk/object_sdk.js

```diff
@@ -163,15 +163,14 @@ class ObjectSDK {
         try {
             if (bucket.namespace && bucket.namespace.read_resources && bucket.namespace.write_resource) {
 
-                // TODO: NAMESPACE_CACHE_DEV is temporarily used for manual override
-                bucket.namespace.cache = (process.env.NAMESPACE_CACHE_DEV === 'true');
-
-                if (bucket.namespace.cache) {
+                dbg.log0('_setup_bucket_namespace', bucket.namespace);
+                if (bucket.namespace.caching) {
                     return {
                         ns: new NamespaceCache({
                             namespace_hub: this._setup_single_namespace(_.extend({}, bucket.namespace.write_resource)),
                             namespace_nb: this.namespace_nb,
                             active_triggers: bucket.namespace.write_resource,
```

Comment:
@guymguym
You should pass the active_triggers that you've received from read_bucket_sdk_info.
We have support for triggers in namespace_nb and namespace_merge (supports s3, blob etc...).
Since namespace_cache is essentially like namespace_merge (not like namespace_s3/blob/etc...), you will need to call the `object_sdk.should_run_triggers` and `object_sdk.dispatch_triggers` methods from the namespace_cache in the relevant paths.

Reply:
I will make the changes.

Reply:
@guymguym @jeniawhite I made the changes by following namespace_merge. Could you review?

Reply:
looks ok to me, but mostly I wonder why we can't move the triggers code up one level to object_sdk around where it calls the namespace functions, and then have it in one place for all namespaces.

Reply:
What are the namespaces supporting active_triggers? Are they nb, merge and cache so far? By glancing through the code, it seems that they have different logic in active_triggers, for example,
- nb has trigger in complete_object_upload, but merge does not have trigger in complete_object_upload.
- merge has logic in handling multiple read sources.

Reply:
Found that there is bug in my original changes related to active triggers in delete_multiple_objects(). For now, my opinion is not to move the logic to object_sdk since the code is not much common.

Reply:
OK so for now we can use the same code copied from namespace_merge, but going forward we might be able to have a default code applied by object_sdk and the namespace will override if needed for optimizing specific cases where we already have all the information locally and do not need to fetch it. I am resolving this comment.